app/views/questions/show.html.erb

```diff
@@ -0,0 +1,68 @@
+<h2><%= @question.question_title %></h2>
+<p><%= @question.content %></p>
+
+<% if logged_in? %>
+  <%= form_for @question, url: question_votes_path(@question), method: :post do |f| %>
+    <%= f.hidden_field :user_id, value: current_user.id %>
+    <%= f.submit "upvote" %>
+  <% end %>
+
+  <%= form_for @question, url: question_votes_path(@question), method: :post do |f| %>
+    <%= f.hidden_field :user_id, value: current_user.id %>
+    <%= f.submit "downvote" %>
+  <% end %>
+<% end %>
+
+
+<p>Comments on Question</p>
+<% @question.comments.each do |qc| %>
+  <p><%= qc.content %> by <%= qc.user.username %></p>
+<% end %>
+
+<% if logged_in? %>
+  <p><%= link_to "New Comment", new_question_comment_path(@question.id) %></p>
+<% end %>
+
+  <% if @fav_answer %>
```

Comment:
I would do some of this in the controller. Make @fave_answer and @other_answers and iterate in the view.